storage/pebble/registers.go

```diff
@@ -0,0 +1,93 @@
+package pebble
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/pebble"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// library that implements pebble storage for registers
+type Registers struct {
+	db *pebble.DB
+}
+
+func NewRegisters(db *pebble.DB) (*Registers, error) {
+	return &Registers{
+		db: db,
+	}, nil
+}
+
+// Get returns the most recent updated payload for the given RegisterID.
+// "most recent" means the updates happens most recent up the given height.
+//
+// For example, if there are 2 values stored for register A at height 6 and 11, then
+// GetPayload(13, A) would return the value at height 11.
+//
+// If no payload is found, an empty byte slice is returned.
+func (s *Registers) Get(
+	height uint64,
+	reg flow.RegisterID,
+) ([]byte, error) {
+	iter, err := s.db.NewIter(&pebble.IterOptions{
```

Comment:
is the height check going to be added in another PR?

If so, please add a TODO here